prototype/chess.cpp

```diff
@@ -35,7 +35,8 @@ enum Piece : uint8_t {
 };
 
 enum Space {
-  A1,
+  START = 0
+  A1 = 1

```

Comment:
I'm confused, why is this being changed?

Reply:
I am also confused, not sure how I did that. Reverting those changes